### PR TITLE
add some useful files

### PR DESCRIPTION
**Pull request by boehs**

### CODE REVIEW

**Comment by uncenter** [comment]
no

**Comment by uncenter** [comment]
maybe

File: SECURITY_CONTACTS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+security@npmjs.com
+support@npmjs.com
+jobs@npmjs.com
+whoshiring@npmjs.com
+press@npmjs.com
+@npm_support on twitter
+@npmjs on Twitter
+me
```
Comment (uncenter) [inline SECURITY_CONTACTS:8]: 🤨

Comment (boehs) [reply SECURITY_CONTACTS:8]: i am highly secure and emotionally available

**Comment by PatrickJS** [comment]
🤔 let me review with the board

**Review by uncenter — APPROVED** [review]
lgtm

**Comment by PatrickJS** [comment]
can you add code of conduct

**Review by uncenter — APPROVED** [review]

**Comment by uncenter** [comment]
dont merge until the ci finishes tho

**Comment by boehs** [comment]
> can you add code of conduct

i am not a lawyer ubforutnateyl

**Review by uncenter — APPROVED** [review]

**Pull request merged**